open-api/immich-openapi-specs.json

```diff
@@ -8286,6 +8299,7 @@
           },
           "quotaSizeInBytes": {
             "format": "int64",
+            "minimum": 1,
```

Comment:
Do we want to allow 0 for this?

Reply:
In the docs/UI we specify setting 0 for unlimited quota, and the final quota check also implicitly relies on that.
https://github.com/immich-app/immich/blob/c9a079201a38a4b5d272e8120c6daf7acb30bba5/server/src/services/asset-v1.service.ts#L363

Reply:
Yeah I was thinking the same, but then I wonder if that even was possible in the past since `@IsPositive()` should only allows numbers > 0 lol
<img width="467" alt="Screenshot 2024-04-22 at 21 08 11" src="https://github.com/immich-app/immich/assets/36593685/25e34a21-4a49-49f3-8cde-663b5dcb5930">

Reply:
I think we should make this consistent - either we allow 0 and use that to signify unlimited quota, or we make it explicit that the field being null/undefined means unlimited quota.

Reply:
Yeah I allow 0 now, but I was primarily just confused that it was working before.

Reply:
You set it to null to reset it. This PR just documents the current rules/enforcement in the spec, it doesn't actually change the implementation.

Reply:
`quotaSizeInBytes: quotaSize ? convertToBytes(Number(quotaSize), 'GiB') : null,` the web sends null when the user puts in 0.

Reply:
Ok that works for me!